2024/0740-nge.md

```diff
@@ -1 +1,27 @@
+
+NGE is a web-based tool with a [frontend](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend) developed using [SBB open source Angular components](https://github.com/sbb-design-systems/sbb-angular), providing an interactive graphical interface accessible through a web browser. Much of the logic is implemented in TypeScript on the frontend to ensure performance and interactivity with low latency and maximum responsiveness. The [backend](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend/), implemented with the Spring Framework and Spring Boot mainly persists the data.
+
+NGE, initially developed internal at SBB, became open source in 2024. Similarly, the SNCF’s [Open Source Railway Designer (OSRD)](https://osrd.fr/en/) focuses on microscopic medium and short-term timetabling, capacity analysis, and simulation. In 2024, a productive collaboration began between the two development teams. This collaboration led to contributions from the OSRD team to the NGE project. In return, the OSRD team integrated NGE components into their software. Such a fruitful collaboration would have been unlikely without the OpenRail Association. In an independent project, an open source converter was developed to export data from NGE into timetables for the entire service day in different formats, such as GTFS static or [MATSim](https://matsim.org/) transit schedules.
+
+To build a broad community and develop its full potential, the Netzgrafik-Editor needs additional user groups within transport companies, public administrations, and educational institutions. For timetable planners or students in transport planning, opening the source code is insufficient. To bridge the gap between software developers and end-users, the application was made [freely accessible](https://nge.flatland.cloud) online in 2024 to ensure that it reaches a broader audience. Open-source success relies not just on code availability but also on user engagement and accessibility.
+
+In 2025, the focus is strengthening the community to attract more users and active developerson developing functionalities along the 
+[roadmap](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend/blob/main/ROADMAP.md).
+
+Further information:
+- [Netzgrafik-Editor Frontend on GitHub](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-frontend)
+- [Netzgrafik-Editor Backend on GitHub](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-backend/)
+- [Netzgrafik-Editor Converter on GitHub](https://github.com/SchweizerischeBundesbahnen/netzgrafik-editor-converter)
+- [Online Application](https://nge.flatland.cloud) operated in the cloud of the [Flatland Association](https://www.flatland-association.org/home)
```

Comment:
Application sounds a bit like applying for something. Maybe something like this:

```suggestion
- [Online Demo Instance](https://nge.flatland.cloud) operated in the cloud of the [Flatland Association](https://www.flatland-association.org/home)
```